requirements.txt

```diff
@@ -0,0 +1,2 @@
+python-arango
```

Comment:
Do you want to ping those to specific versions? E.g., by `pip freeze -r requirements.txt > stable-req.txt`

Reply:
yes, that makes sense.